diff --git a/build/app/navigation/index.d.ts b/build/app/navigation/index.d.ts
index 09991a07a6cf0ef57edfc4da937b433474d19009..fd1ce0f6887810226068f4f2b5d8e02deeb43db3 100644
--- a/build/app/navigation/index.d.ts
+++ b/build/app/navigation/index.d.ts
@@ -2,5 +2,6 @@ export * from './use-router';
 export * from './use-params';
 export * from './use-search-params';
 export * from './use-pathname';
+export * from './use-link';
 export { default as useUpdateSearchParams } from './use-update-search-params';
 //# sourceMappingURL=index.d.ts.map
\ No newline at end of file
diff --git a/build/app/navigation/index.js b/build/app/navigation/index.js
index 70708a4f564e402dfe3b68203000d9fc86e9d9d8..a3586a64d25971d79b88cf2ae619678008ced25a 100644
--- a/build/app/navigation/index.js
+++ b/build/app/navigation/index.js
@@ -2,5 +2,6 @@ export * from './use-router';
 export * from './use-params';
 export * from './use-search-params';
 export * from './use-pathname';
+export * from './use-link';
 export { default as useUpdateSearchParams } from './use-update-search-params';
 //# sourceMappingURL=index.js.map